middleware/src/middleware.go

```diff
@@ -261,6 +303,11 @@ func (middleware *Middleware) BackupSysconfig() (response rpcmessages.ErrorRespo
 		}
 	}
 
+	//BackupSysconfig is the last call in the base setup. Once it is completed successfully, the setup is done.
+	err = middleware.redisClient.SetString(redis.BaseSetupDone, "1")
+	if err != nil {
+		log.Println("redis failed to set the base setup done flag. This is not a critical error, but the setup will be repeated again")
+	}
```

Comment:
This should be moved eventually. I opend an issue for it https://github.com/shiftdevices/bitbox-base-internal/issues/312